### ISSUE TITLE
Feedback from Colleage (Brian)

### ISSUE THREAD

**Issue by psmacmur** [danielabar/globi-proto#28]
Documenting all the feedback from my colleague here, then will split this up into issues as time permits and after some team discussion:

## Overall

* Doesn't seem to convey insight as to biotic interactions: how things are interconnected, scope of interaction networks, etc.  
* Try to support learning, e.g. by tracking the path people take and supporting backtracking.
* Looks like a database browser for a specific domain, in which case, include more metadata at each step.
* Consolidate the network view into the cards view

## Explore

* Simplify the dropdown.  
  * Consolidate into a handful of categories, then use different visual encodings to show specific relationships within one view (cards/network/etc)

*  Flip isn't useful
  * Put everything on the front 
  * use 'i' for info instead of globe icon for 'Site'
  * just use icons, to get a few onto the front
  * avoid dropdowns

* Include more metadata: 
  * Categorize species.  e.g. is it plant, animal, marine, etc.  
      * group cards
  * what interactions does it participate in?
    * how many things does tuna eat?  
    * how many things eat tuna?
    * how many parasites does it have? 

* show links between multiple items.
  * perhaps a compare mode where you check off several cards & click to get their network
  * or integrate the network view into this view

## Network Graph
* Doesn't seem like the right viz for the goals
* Color needs to encode something; horizontal space is already encoding the levels
* Network is really weird
* Maybe draw the back arrows for the leaf nodes?
* Crustacea
  * Looks like a category a child of a species (tuna)
  * If it is a category, how many items in the category?

## Play

* What is Play?
* Looks promising

**Comment by danielabar** [comment]
Sigh, this is disheartening "Doesn't seem like the right viz for the goals" and "Network is really weird"

**Comment by psmacmur** [comment]
Yeah those aren't exactly his words, well at least the first one isn't, but
he clearly thought what we had was pretty useless.

Oh well...

Peter

**Comment by danielabar** [comment]
This is what I was worried about, there's no way we have time at this point to totally change direction.

**Comment by psmacmur** [comment]
This is true. We can clean up a few easy things that come up in all the
feedback, and that's about it.  E.g. maybe redo the card layout to be 1
side again, tweak the colors in the network, etc., but there really isn't
time for the overhaul it'd need at this point IRL.

Guess we'll have to stick to coding... so much for our budding careers as
information designers or data scientists.

Peter

**Comment by danielabar** [comment]
Yeah, that sounds reasonable to do small tweaks.
As for the infovis, guess that's all part of the course, learning.
On the bright side, the feedback I got from Sean is more actionable, and less soul destroying :-p

**Comment by psmacmur** [comment]
Hmm I doing this mockup of the "Interaction" tab that we'd talked about
upgrading the map tab too, and now I'm thinking we should just drop it and
go with Stefan's mockup that puts the map in the network tab.  Especially
based on the feedback that it's not integrated enough.   Is that feasible?

Peter

**Comment by danielabar** [comment]
Yeah let's pretend the Map view as a separate tab no longer exists.
And put the map below the graph in Network tab.
Then we won't need a separate Interaction tab.

In fact with Sean's idea, the main/home view wouldn't have any navigation tabs at all (except possibly an About tab).

Then when user somehow selects to view the network for the Source taxon, then they get taken to the Network page with an alternate view of the nav. This nav would display the Source taxon name,  a link back to Explore so they can explore other species.

The good news about this is we already have the scaffolding in place to have different nav views for each main content switch, so that part in and of itself should be easy.

**Comment by psmacmur** [comment]
Brilliant.

In interest of not throwing babies out with bath water, I mightn't delete
the tabs per se yet.  I mean, delete Map and Play, but the ones we use can
be there, just to show up front what all we can do.   For now, maybe each
with a default for demo purposes or something, if that's easy.

Some of Brian's suggestions were actionable, too.   Based on his
recommendation of avoiding dropdowns, and in interest of not throwing out
my entire saturday's work, i'm wondering if "flip" becomes "more
interactions" with only the selected (from the form) interaction on the
front (e.g. eats) along with the other links (Site, Network), with a longer
interaction list on the back.

Wondering if we can also narrow down the interaction list, since some don't
seem to have any data anymore.  That might be a case of asking Jorrit to
remove deprecated interactions from the API?  At any rate, only our
favourites should go onto the backs of the cards (in this new idea)... does
Stefan's analysis give any indication of which interactions are well
represented?

Brian did also ask about the About tab, before seeing it, although he had
no idea what should be there.


Peter

**Comment by danielabar** [comment]
Map and Play are gone for sure.

About can stay, that should be a constant in all the nav versions we will have. We'll think of some appropriate content later. At the very least some links to the globi/eol, api stuff, maybe version number, link to source on github.

Network should also go, because Sean said it was really confusing, given that he's selected a taxon in Explore, if he clicks directly to Network, it shows something else completely. 

He also said it was really confusing to have the interactions (like eats, eaten by etc.) both in the dropdown on explore page and in the cards. However, it should be possible from a card, to "select" it as the new search point.

As for the interaction dropdown list, we could also ask Stefan to find what are the most frequent ones with results, and just manually populate the dropdown list with the common ones. A risk is if they names change in the API, then it won't work anymore. Yes we could also ask Jorrit if its possible for the API to return the most "populous" ones, not sure if that's a valid word.

So back to the cards (yes the flipping is really cool), so here's what should go on there, let's see how to distribute between front and back.

I'm also speculating on two different kinds of cards: A card with the taxon you're searching on, and the interaction results cards.

Taxon Card (this is the subject of your search)
* Thumbnail image
* Scientific name
* Common name
* Link to eol site (external url)
* Clickable icon to network view (internal)

Interaction Results Card:
* Thumbnail image
* Scientific name
* Common name
* Link to eol site (external url)
* Clickable search icon to make this taxon the new search
